### ISSUE TITLE
The OAuth Personal Token could not be verified

### ISSUE THREAD

**Issue by thananit** [envato/wp-envato-market#57]
Hello,

I installed your plugin and activated and registered your App and created a new token and copy an Envato API Personal Token fill in a token setting and test API connecting.
But it did not work.
The message alert;
"The OAuth Personal Token could not be verified. Please check that the Token has been entered correctly and has the minimum required permissions."

How to slove this issue?
Please advise me.

Thank you,
Thananit

**Comment by thananit** [comment]
Additional information;
My App area had a message;
"Apps you've granted access
You haven't granted access to any app yet. Once you do, you'll be able to revoke the apps' permissions here."

I don't understand about it.
Please advise.

Thank you,
Thananit

**Comment by brandonfiredesign** [comment]
I'm having the same problem.  Any updates or suggestions?  Followed the directions to the letter. So what's up?

Thanks for any help on this!!

**Comment by dtbaker** [comment]
There have been some recent API changes. Hey please let us know if this is still an issue and we can look into it further.

**Comment by Evelyn7** [comment]
Hi+
I downloaded the zip file from your website on 02/26/18, then removed the old plugin and installed the new zip file via the back-end Plugins > add new > upload. I then activated - and it still did not work. It did show an "update" button/link, however, when I clicked that nothing happened.

WP v 4.9.4
Envato Plugin v. 1.0.0-RC2

So, is the zip file on your website the "new" (fixed) version of the plugin? If so, why is it not working as expected? If not, where do I find the correct version?
Thanks!
